docs/unified-test-documentation/dasharo-compatibility/69-thunderbolt-ME.md

```diff
@@ -0,0 +1,1939 @@
+# Dasharo Compatibility: USB-C/Thunderbolt support with charging and display
+
+## Test cases common documentation
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../generic-test-setup.md#firmware).
+1. Proceed with the
+    [Generic test setup: OS installer](../generic-test-setup.md#os-installer).
+1. Proceed with the
+    [Generic test setup: OS installation](../generic-test-setup.md#os-installation).
+1. Proceed with the
+    [Generic test setup: OS boot from disk](../generic-test-setup.md#os-boot-from-disk).
+
+## TMD001.001 USB Type-A charging capability with ME disabled
```

Comment:
could we add these tests as variants of existing tests in suite UTC? i.e. `UTC001.002  USB Type-A charging capability (ME disabled)`

Reply:
sure: https://github.com/Dasharo/docs/pull/874/commits/c3480c6cb5860476e9ead31dcd82d743120fd53e

Reply:
Thanks